packages/web/src/components/gcds-select/gcds-select.tsx

```diff
@@ -338,20 +361,28 @@ export class GcdsSelect {
           <select
             {...attrsSelect}
             id={selectId}
-            name={selectId}
-            onBlur={e => this.onBlur(e)}
-            onFocus={e => this.onFocus(e)}
+            onBlur={() => this.onBlur()}
+            onFocus={() => this.onFocus()}
             onChange={e => this.handleChange(e)}
             aria-invalid={hasError ? 'true' : 'false'}
-            ref={element => (this.shadowElement = element as HTMLElement)}
+            ref={element => (this.shadowElement = element as HTMLSelectElement)}
           >
             {defaultValue ? (
               <option value="" disabled selected>
                 {defaultValue}
               </option>
             ) : null}
-
-            <slot></slot>
+            {options.map(opt => {
+              const selected = {};
+              if (opt.hasAttribute('selected')) {
+                selected['selected'] = true;
+              }
```

Comment:
Instead of adding an empty object and conditionally adding properties to it, you can use a ternary operator to conditionally add the `selected` attribute.

`const selected = opt.hasAttribute('selected') ? { selected: true } : null;`